### PR TITLE
DO NOT MERGE WITHOUT DISCUSSION

### PR DESCRIPTION
**Pull request by stmuk**
Given that tempers are high about this discussion I don't suggest merging this (or anything similar) right now.

I think do need a one mild reference to "Raku" on the home page at some point as something extra and not replacing any current perl 6 use. Perl 6 as a name isn't going away.

It seems to me that "nickname" fits in with alias or stage name and "little sister".

### CODE REVIEW

File: source/index.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h1>The <b>Perl&nbsp;6</b> Programming Language</h1>
       <p>Hi, my name is Camelia.  I'm the spokesbug for Perl&nbsp;6,
         the plucky little sister of <a href="https://www.perl.org/">Perl&nbsp;5</a>.
         Like her world-famous big
-        sister, Perl&nbsp;6, now nicknamed "Raku", intends to carry forward the high ideals of
+        sister, Perl&nbsp;6, now nicknamed by some to be "Raku", intends to carry forward the high ideals of
```
Comment (AlexDaniel) [inline source/index.html]: What about something neutral, like:

> Perl 6, also known as “Raku”, …

Or maybe it is not more neutral?

Comment (stmuk) [reply source/index.html]: My intention in saying "nicknamed by some" was to address the fact that not all people like the nickname. It was an attempt to be neutral or at least more honest. 

It's clearly not wise to merge anything right yet and to take time over over the exact wording.

**Comment by JJ** [comment]
What do we do with this now?

**Comment by AlexDaniel** [comment]
@JJ restart the discussion using the [problem-solving](https://github.com/perl6/problem-solving/). It needs to be done in a structured way, strictly like the document suggests.

**Comment by AlexDaniel** [comment]
To clarify a bit, first you need to start a ticket that clearly defines the problem. Shortly it is “the language has two names, but only one is reflected on the website”, but that probably needs to be elaborated a bit more.

**Pull request closed without merging**